src/pages/libs/draftjs-utils/inline.ts

```diff
@@ -225,3 +228,53 @@ export const getSelectionCustomInlineStyle = (
   }
   return {};
 };
+
+interface ChangeOriginalStyleToNeweStyle {
+  editorState: EditorState;
+  originalStyle: StyleKeyType;
+  newStyle?: StyleKeyType;
+  startOffset?: number;
+  endOffset?: number;
+}
+
+export const changeOriginalStyleToNeweStyle = ({
+  editorState,
+  originalStyle,
+  newStyle,
+  startOffset,
+  endOffset,
+}: ChangeOriginalStyleToNeweStyle): EditorState => {
+  const selectionState = editorState.getSelection();
+  const newSelection = selectionState.merge({
+    anchorOffset: startOffset,
+    focusOffset: endOffset,
+  });
+
+  const editorStateWithNewSelection = EditorState.forceSelection(
+    editorState,
+    newSelection,
+  );
+
+  const editorStateWithToggleOriginalStyle = RichUtils.toggleInlineStyle(
+    editorStateWithNewSelection,
+    originalStyle,
+  );
+
+  let editorStateWithNewStyle;
+
+  if (newStyle != null) {
+    editorStateWithNewStyle = RichUtils.toggleInlineStyle(
+      editorStateWithToggleOriginalStyle,
+      newStyle,
+    );
+  } else {
+    editorStateWithNewStyle = editorStateWithToggleOriginalStyle;
+  }
```

Comment:
```suggestion
  if (newStyle == null) {
    editorStateWithNewStyle = editorStateWithToggleOriginalStyle;
  } else {
    editorStateWithNewStyle = RichUtils.toggleInlineStyle(
      editorStateWithToggleOriginalStyle,
      newStyle,
    );
  }
```
p4: if-else 처리 할 때 if에 not 이 안 붙어있는게 개인적으로 좀 더 읽기 편하더라구여